src/ontology/cl-edit.owl

```diff
@@ -23391,6 +23392,16 @@ SubClassOf(obo:CL_0011029 obo:CL_0000151)
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0042151))
 SubClassOf(obo:CL_0011029 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_6073))
 
+# Class: obo:CL_0011031 (monocycte-derived dendritic cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32733468") obo:IAO_0000115 obo:CL_0011031 "A dendritic cell subset that develops from monocytes monocytes upon stimulation in the circulation that are involved in inflammation and infection.")
+AnnotationAssertion(oboInOwl:created_by obo:CL_0011031 <http://orcid.org/0000-0001-5208-3432>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0011031 "BTO:0002900")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0011031 "MDDC cell")
+AnnotationAssertion(rdfs:label obo:CL_0011031 "monocycte-derived dendritic cell")
+EquivalentClasses(obo:CL_0011031 ObjectIntersectionOf(obo:CL_0000451 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000576)))
+SubClassOf(obo:CL_0011031 obo:CL_0000451)
```

Comment:
Is the subClassOf assertion or can it be inferred?

Reply:
We have always put 'develops from' assertions as subclass axioms, not equivalence axioms.

Reply:
in addition?

Reply:
if we don't have the equiv axiom, the parent 'dendritic cell' and 'myeloid leukocyte' is not inferred

Reply:
Given the name, it totally makes sense for develops_from to be in the equivalence axiom.  No need to repeat in subClassOf